tests/other/test_parseTools.c

```diff
@@ -0,0 +1,13 @@
+#include <limits.h>
+#include <stdio.h>
+#include <inttypes.h>
+
+void test_makeGetValue(uint32_t* ptr);
+
+int main() {
+  uint32_t num = UINT32_MAX;
```

Comment:
Maybe make the value `0x12345678` or such, so the test also verifies we get the right byte(s) and in the right order?

Reply:
Good idea!